.github/workflows/ql-for-ql-build.yml

```diff
@@ -0,0 +1,160 @@
+name: Run QL for QL
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  queries:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Cache queries
+        id: cache-queries
+        uses: actions/cache@v2
+        with:
+          path: ${{ runner.temp }}/query-pack.zip
+          key: ${{ runner.os }}-queries-${{ hashFiles('**/*.ql*') }}
```

Comment:
`runner.os` should not have a say in how queries are compiled. Can we remove that part of the key?